fe/src/components/ColorMode/ColorMode.tsx

```diff
@@ -0,0 +1,16 @@
+import { FaMoon, FaSun } from 'react-icons/all';
+import { HStack, IconButton, useColorMode } from '@chakra-ui/react';
+
+export default function ColorMode(): JSX.Element {
+  const { colorMode, toggleColorMode } = useColorMode();
+  return (
+    <HStack spacing={12} ml="auto" title={colorMode === 'light' ? 'Switch to dark theme' : 'Switch to light theme'}>
```

Comment:
Make it so that ColorMode component only has IconButton. HStack should stay on Header